Formula/nuraft.rb

```diff
@@ -18,10 +19,11 @@ class Nuraft < Formula
 
   depends_on "cmake" => :build
   depends_on "asio"
-  depends_on "openssl@1.1"
+  depends_on "openssl@3"
 
   def install
-    system "cmake", "-B", "build", *std_cmake_args
+    # We override OPENSSL_LIBRARY_PATH to avoid statically linking to OpenSSL
+    system "cmake", "-S", ".", "-B", "build", *std_cmake_args, "-DOPENSSL_LIBRARY_PATH="
```

Comment:
Relying on fallback behavior to just use `-lssl -lcrypto`. If Homebrew's copy is detected, then it will use static libraries. See https://github.com/eBay/NuRaft/blob/v1.3.0/CMakeLists.txt#L130-L138
```cmake
    if (NOT OPENSSL_LIBRARY_PATH)
        message(STATUS "Use system's side OpenSSL library")
        set(LIBSSL ssl)
        set(LIBCRYPTO crypto)
    else ()
        message(STATUS "Open SSL library path: ${OPENSSL_LIBRARY_PATH}/libssl.a")
        set(LIBSSL ${OPENSSL_LIBRARY_PATH}/libssl.a)
        set(LIBCRYPTO ${OPENSSL_LIBRARY_PATH}/libcrypto.a)
    endif ()
```